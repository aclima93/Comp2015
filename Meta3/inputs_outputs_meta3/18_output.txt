Line 5, col 14: Symbol A already defined
