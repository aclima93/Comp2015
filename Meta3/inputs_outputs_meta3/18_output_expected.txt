===== Outer Symbol Table =====
boolean	_type_	constant	_boolean_
integer	_type_	constant	_integer_
real	_type_	constant	_real_
false	_boolean_	constant	_false_
true	_boolean_	constant	_true_
paramcount	_function_
program	_program_

===== Function Symbol Table =====
paramcount	_integer_	return

===== Program Symbol Table =====
a	_integer_
b	_integer_
c	_integer_
x	_real_
spj	_function_
gcd	_function_

===== Function Symbol Table =====
spj	_integer_	return
w	_integer_	param

===== Function Symbol Table =====
gcd	_integer_	return
d	_integer_	param
e	_integer_	param
